Sources/VitalHealthKit/Extensions/Date.swift

```diff
@@ -1,7 +1,8 @@
 import Foundation
 
+/// UTC Gregorian calendar for standard statistical queries.
 let vitalCalendar: Calendar = {
-  var calendar = Calendar.autoupdatingCurrent
+  var calendar = Calendar(identifier: .gregorian)
```

Comment:
What potential issues might we have here with this change?

Reply:
Very unlikely any:
(1) since using an alternative calendar is somewhat rare, especially for our current audiences.
(2) we use `Calendar` only for the date maths, not for localization into user-facing strings. So losing the device locale auto-tracking is irrelevant.